src/SourceBrowser.Site/Scripts/upload.js

```diff
@@ -0,0 +1,29 @@
+﻿$(document).ready(function () {
+    resetUI();
+    registerButtonClickEvent();
+});
+
+function registerButtonClickEvent() {
+    getButtonFromDom().click(prepareForChange);
+}
+
+function prepareForChange() {
+    getButtonFromDom().prop("disabled", true);
+    getMessageDivFromDom().css("visibility", "visible");
+}
+
+function resetUI() {
+    getButtonFromDom().prop("disabled", false);
+    getMessageDivFromDom().css("visibility", "hidden");
```

Comment:
A fair point.  I was trying to avoid too much in the way of in-line styling in anticipation of adding a .css eventually.  I'll revisit this!